tactile_merger/include/tactile_merger/merger.h

```diff
@@ -42,8 +42,8 @@ class Merger
 	template <typename Iterator>
 	void update(const ros::Time &stamp, const std::string &channel,
 	            Iterator begin, Iterator end);
-	tactile_msgs::TactileContacts getAverageContacts();
-	tactile_msgs::TactileContacts getAllContacts();
+	tactile_msgs::TactileContacts getGroupContacts();
```

Comment:
Can we finish this? @guihomework, I know (from offline communication) that you want to keep _average_ as part of the function name. What about `getGroupAveragedContacts()`?

Reply:
getGroupAveragedContacts is fine for me.